docker/Makefile

```diff
@@ -1,5 +1,5 @@
 # Base image to use for the docker build.
-BASE_IMAGE ?= 'nvidia/cuda:11.3.1-cudnn8-runtime-ubuntu20.04'
+BASE_IMAGE ?= 'nvidia/cuda:11.3.1-cudnn8-devel-ubuntu20.04'
```

Comment:
We really need devel? 😢  that makes our images much larger

Reply:
Yup, unfortunately we need it for A100s on GCP. See https://github.com/mosaicml/composer/blob/bf0bd3b3fa0bab7f3dd06cf7a9b1d55e7a8cc31b/docker/pytorch/Dockerfile#L36